src/metabase/sync_database/analyze.clj

```diff
@@ -68,6 +68,18 @@
       (and (nil? (:special_type field))
            (pos? (count distinct-values))) (assoc :special-type :type/Category))))
 
+(def histogram-type? #{:type/Latitude :type/Longitude})
+(def histogram-value? (comp histogram-type? :special_type))
+
+(defn test:histogram-fields-with-min-max
+  "Add min/max and special-type values for Longitude/Latitude types"
+  [field field-stats]
+  (let [[max-val min-val] (queries/field-max-min field)]
+    (assoc field-stats
+      :min-value min-val
+      :max-value max-val
+      :special-type (:special_type field))))
```

Comment:
I found that I needed to include special type when I sniffed out the latitude/longitude fields. Is this being populated elsewhere and I missed it?